week3/express.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
```

Comment:
I would change the name of this file to entry points or api entry. But this is a great clean entry point for your api. Very nice. =)